tests/specs/tests.spec.ts

```diff
@@ -66,6 +68,41 @@ describe('tests', () => {
         expect(actualHeaders2!.source).toEqual('test-service2');
     });
 
+    it('should produce and consume with regex patterns', async () => {
+        const callId = await mockHttpTarget('/consume', 200);
+
+        await produce('http://localhost:6000/produce', [
+            {
+                topic: 'lol.bar.baz',
+                key: 'thekey',
+                value: {data: 'foo'},
+                headers: {eventType: 'test1', source: 'test-service1'},
+            },
+        ]);
+        await delay(1000);
+        await produce('http://localhost:6000/produce', [
+            {
+                topic: 'bar',
+                key: 'thekey',
+                value: {data: 'bar'},
+                headers: {eventType: 'test2', source: 'test-service2'},
+            },
+        ]);
+        await delay(1000);
+
+        const {hasBeenMade, madeCalls} = await fakeHttpServer.getCall(callId);
+        expect(hasBeenMade).toBeTruthy();
+        expect(madeCalls.length).toBe(2);
+        const actualHeaders1 = JSON.parse(madeCalls[0].headers['x-record-headers']);
```

Comment:
can you assert actualHeaders1 and then actualHeaders2? with space between them
will make it easier to read